### PR TITLE
Create andhavarapu_balu.json

### PR DESCRIPTION
**Pull request by JustTheCoolest**

### CODE REVIEW

File: src/data/andhavarapu_balu.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Andhavarapu Balu",
+    "branch": "Computer Science and Engineering",
+    "year": "2022 admitted",
```
Comment (lem0n4id) [inline src/data/andhavarapu_balu.json]: Change to "2nd"

File: src/data/andhavarapu_balu.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Andhavarapu Balu",
+    "branch": "Computer Science and Engineering",
```
Comment (lem0n4id) [inline src/data/andhavarapu_balu.json]: Change to CSE

**Review by lem0n4id — CHANGES_REQUESTED** [review]

**Comment by JustTheCoolest** [comment]
@lem0n4id 
Made the requested changes, please pull

**Review by lem0n4id — APPROVED** [review]
LGTM

**Pull request merged**